python/lsst/ctrl/bps/transform.py

```diff
@@ -589,6 +618,11 @@ def create_generic_workflow(config, cqgraph, name, prefix):
                                                   job_access_remote=True,
                                                   job_shared=True))
 
+    # Cache pipetask specific or more generic job values to minimize number
+    # on config searches.
+    cached_job_values = {}
+    pipetask_values = {}
```

Comment:
Just out of curiosity, why `cached_job_values`, but not `cached_pipeline_values`? Judging from the comment, it looks like both these dictionaries will be used as caches.

Reply:
I remember going back and forth on that, but can't remember why I landed not "cached".  So I switched pipelines_values to match (cached_pipeline_values).